addons/tmf_loadouts/loadouts/US_USMC_1985.hpp

```diff
@@ -0,0 +1,493 @@
+		"ItemRadio",
+		"ItemWatch"
+	};
+
+	// These are put into the backpack
+	backpackItems[] = {};
+
+	// Unit traits, see https://community.bistudio.com/wiki/setUnitTrait
+	traits[] = {};
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	headgear[] = {
+		"CUP_H_PASGTv2_desert"
+	};
+	primaryWeapon[] = {
+		"CUP_arifle_M16A2"
+	};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_Stanag"),
+		LIST_2("CUP_30Rnd_556x45_Stanag_Tracer_Red"),
+		LIST_2("HandGrenade"),
+		"SmokeShell"
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	primaryWeapon[] = {"CUP_arifle_M16A2_GL"};
+	vest[] = {"usm_vest_pasgtdes_lbe_gr_m"};
+	magazines[] += {
+		LIST_11("CUP_1Rnd_HEDP_M203"),
+		LIST_2("1Rnd_Smoke_Grenade_shell")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_Stanag"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+	items[] = { MEDICAL_R ,"ACE_Flashlight_MX991"};
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	backpack[] = {"CUP_B_AlicePack_Khaki"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"usm_pack_alice"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	primaryWeapon[] = {"CUP_smg_MP5A5"};
+	scope[] = {};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_9x19_MP5"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	items[] += {"ACE_MapTools"};
+	magazines[] += {
+		LIST_2("CUP_1Rnd_StarFlare_Red_M203"),
+		LIST_2("CUP_1Rnd_SmokeRed_M203")
+	};
+	linkedItems[] += {"ACE_Yardage450"};
+};
+
+class sl : ftl {
+	displayName = "Squad Leader";
+	vest[] = {"usm_vest_pasgtdes_lbe_rmp_m"};
+	backpack[] = {"CUP_B_AlicePack_Khaki"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	magazines[] += {
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"usm_bdu_8point_des_m"};
+	linkedItems[] = {"ItemGPS"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	backpack[] = {"usm_pack_alice_prc119"};
+	backpackItems[] = { FAC_GEAR };
+	linkedItems[] += {
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	primaryWeapon[] = {"CUP_lmg_M249_E2"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	vest[] = {"usm_vest_pasgtdes_lbe_mg_m"};
+	backpack[] = {"CUP_B_AlicePack_Khaki"};
+	magazines[] = {
+		LIST_3("CUP_200Rnd_TE4_Red_Tracer_556x45_M249"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpack[] = {"usm_pack_200rnd_556_bandoliers"};
+	Items[] += {"ACE_EntrenchingTool"};
+	linkedItems[] += {"Binocular"};
+};
+
+class rat : r {
+	displayName = "Rifleman (AT)";
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
+	secondaryWeapon[] = {"CUP_launch_Mk153Mod0"};
```

Comment:
MAT weapon

Reply:
Replaced with the AT4 (adopted in 1982, iirc).